bin/build

```diff
@@ -18,10 +18,6 @@
 bin_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 at_home=$( cd "$( dirname "$bin_dir" )" && pwd )
 
-if [ -f "$at_home/conf/env.sh" ]; then
-  conf="$at_home/conf/env.sh"
-else
-  conf="$at_home/conf/env.sh.example"
-fi
-echo "Using $conf for setup"
-source $conf
+env_sh="$at_home/conf/env.sh"
+echo "Using $env_sh for setup"
+source $env_sh
```

Comment:
What happens if the files does not exist?

Reply:
The file always exists unless you deleted it from the repo after checking it out.